dockyard/common/image/image.py

```diff
@@ -0,0 +1,34 @@
+from dockyard.common import base, link
+from dockyard.common import utils
+
+class Image(object):
+    def __init__(self):
+        self.rest_client = base.RESTClient()
+
+    def list(self, _id=None):
+        return utils.dispatch_get_request(protocol='http', url='/images/json').data
+
+    def history(self, _id=None):
+        url='/images/%s/history' % _id
+        return utils.dispatch_get_request(protocol='http', url=url).data
+
+    def search(self, term=None):
+        url='/images/search'
+        query = { "term" : term}
```

Comment:
I am not sure but i think, this can be fetched from the query_string, is it really being required to create dictionary like this.